packages/core/src/components/search-input/search-input.stories.js

```diff
@@ -14,6 +14,7 @@ export const Default = () => `
         id="example"
         class="hds-search-input__input"
         type="text"
+        placeholder="Placeholder text"
```

Comment:
> "Placeholder text"

To be in line with other stories, this should only read "Placeholder".

Reply:
- [ ] Placeholder text -> Placeholder